recipe/meta.yaml

```diff
@@ -17,7 +17,8 @@ source:
   
 build:
   number: 0
-  skip: true # [py2k]
+  skip: true  # [py2k]
+  skip: true  # [win32]
```

Comment:
When making this sort of change, please remember to [re-render]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ).